### ISSUE TITLE
Markdown parser doesn't support fenced code blocks

### ISSUE THREAD

**Issue by drzax** [openaustralia/morph#891]
It's pretty common to put fenced code blocks in readmes on GitHub.

```
// Like this one
```

It would be nice to support this on scraper pages on morph.io. See https://morph.io/drzax/australian-daylight-savings for an example.

**Comment by henare** [comment]
Yeah that's bug. We use the same library as GitHub so we probably just need to update it.

**Comment by henare** [comment]
Oh and thank you for reporting this! :smile:

**Comment by snoblenet** [comment]
@henare I'm happy to pick up this one too, but I won't get started until I hear back from you in response to my latest comments on #320 as I plan to use the same -- seed data, feature specs and pry-rescue.

**Comment by benrfairless** [comment]
I've closed #941 as a duplicate of this issue - it was a help forum report of a README not rendering its formatting, which comes back to the markdown parser not supporting fenced code blocks.

This was carried out with Claude Code 2.1.198 (Claude Fable 5) after being reviewed by me.
